README.md

```diff
@@ -1,13 +1,234 @@
-OpenPipeline
-================
+# OpenPipeline
 
-<!-- README.md is generated by running 'quarto render README.qmd' -->
+OpenPipeline are extensible single cell analysis pipelines for reproducible and large-scale single cell processing using Viash and Nextflow.
 
-Extensible single cell analysis pipelines for reproducible and
-large-scale single cell processing using Viash and Nextflow.
+[![GitHub](https://img.shields.io/badge/GitHub-openpipelines--bio%2Fopenpipeline-blue.svg)](https://github.com/openpipelines-bio/openpipeline)
+[![GitHub
+License](https://img.shields.io/github/license/viash-hub/demultiplex.svg)](https://github.com/viash-hub/demultiplex/blob/main/LICENSE)
+[![GitHub
+Issues](https://img.shields.io/github/issues/openpipelines-bio/openpipeline.svg)](https://github.com/viash-hub/demultiplex/issues)
+[![Viash
+version](https://img.shields.io/badge/Viash-v0.9.4-blue)](https://viash.io)
```

Comment:
Can we fill in the following information from the _viash.yaml in the same way it was done with the qmd before? 

* Viash version
* package name
* repository name

Because:

* I see this already links partially to demultiplexing
* The viash version will be annoying to maintain

Reply:
Done!